src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -166,13 +167,27 @@ const writeAccount = async function writeAccount(req: Request, res: Response): P
     }
   }
 
+  const stripeStatementDetails: {[key: string]: string} = {
+    payoutStatementDescriptor: '',
+    statementDescriptor: ''
+  } 
+
+  if (account.provider === 'stripe'){
+    const stripeAccountDetails = await stripe.accounts.retrieve(account.credentials);
+    stripeStatementDetails.payoutStatementDescriptor =  stripeAccountDetails.payout_statement_descriptor
+    stripeStatementDetails.statementDescriptor =  stripeAccountDetails.statement_descriptor
+  }
+
   // note payment_provider is not returned in the object returned from createAccount
   res.render('gateway_accounts/createSuccess', {
     account: createdAccount,
     linkedService,
     gatewayAccountIdDerived,
     provider: account.provider,
-    isLive: account.isLive()
+    isLive: account.isLive(),
+    isStripe: account.provider === 'stripe' ? true : false,
+    stripeStatementDetails,
+    baseGoLiveUrl: services.SELFSERVICE_URL
```

Comment:
Recommend clarifying what this is
```suggestion
    selfServiceBaseUrl: services.SELFSERVICE_URL
```

Reply:
Done